webapp/src/service/billingApiSchema.generated.ts

```diff
@@ -1394,6 +1407,49 @@ export interface operations {
       };
     };
   };
+  addUsageItemsToInvoiceAndFinalizeIt: {
```

Comment:
As discussed in person, I would put the forgotten changes into a separate PR, but it is just a matter of taste.